### DISCUSSION TITLE
Two Suggestions - docker with all associated servers packed in, rerankers library

### DISCUSSION THREAD

**Discussion by phren0logy** [KruxAI/ragbuilder#40] [Ideas]
# 1. Docker additions
I saw that n8n recently published the following repo: https://github.com/n8n-io/self-hosted-ai-starter-kit

It has a Docker compose file with:
- PostgresQL
- Qdrant
- Ollama
- n8n (obviously, as it's their thing)

Your Docker image already included Neo4j, but something more comprehensive like this would be even better. For my part, I've been experimenting with the Kuzu graph database, and found it to be very interesting as it doesn't need a server at all.

# 2. Reranker library

This library: https://github.com/AnswerDotAI/rerankers looks like it might be an easy way to have RagBuilder try various reranking options

Thanks for considering this feedback.

**Comment by aravind10x** [comment]
@phren0logy, fantastic suggestions! Thank you! We will add it to the roadmap. Also, you are welcome to contribute :) Happy to share the detailed roadmap with you if you'd like to pick something.
